### PR TITLE
VTCCA-4041: migrated the addUserToGG page

### PR DESCRIPTION
**Pull request by habeeb01**

### CODE REVIEW

**Comment by reddy-yattapu** [comment]
:+1:

File: app/views/mainLayout.scala.html [hunk 1]
```diff
@@ -33,7 +34,7 @@
         backUri: Option[String] = None,
         headExtra: Html = Html(""),
         backLinkClass: String = "govuk-back-link",
-        hasErrors: Boolean = false
+        hasErrors: Boolean = false, showTopNavigation: Boolean = true
```
Comment (darrell-plant) [inline app/views/mainLayout.scala.html:37]: newline before `showTopNavigation` would look better

**Comment by darrell-plant** [comment]
LGTDz 👍 (minor comment)

File: app/views/mainLayout.scala.html (hunk 1 above)
Comment (habeeb01) [reply app/views/mainLayout.scala.html]: addressed @darrell-plant

**Pull request merged**